ownCloud/Theming/ThemeCollection.swift

```diff
@@ -73,9 +73,29 @@ enum ThemeCollectionStyle : String, CaseIterable {
 	}
 }
 
+enum ThemeCollectionInterfaceStyle : String, CaseIterable {
+	case dark
+	case light
+	case unspecified
+
+	@available(iOS 12.0, *)
```

Comment:
@felix-schwarz did you mean iOS 13.0?

Reply:
It's there mostly because of iOS 13, yeah, but since `UIUserInterfaceStyle` is available since iOS 12, that's what I use here.